agent_pool_integration_test.go

```diff
@@ -200,6 +200,40 @@ func TestAgentPoolsUpdate(t *testing.T) {
 		assert.Nil(t, w)
 		assert.EqualError(t, err, ErrInvalidAgentPoolID.Error())
 	})
+
+	t.Run("when updating allowed-workspaces", func(t *testing.T) {
+		kBefore, kTestCleanup := createAgentPool(t, client, orgTest)
+		defer kTestCleanup()
+
+		workspaceTest, workspaceTestCleanup := createWorkspace(t, client, orgTest)
+		defer workspaceTestCleanup()
+
+		kAfter, err := client.AgentPools.Update(ctx, kBefore.ID, AgentPoolUpdateOptions{
+			Name: String(kBefore.Name),
+			AllowedWorkspaces: []*Workspace{
```

Comment:
🤔 How are the tests passing when the allowed-workspaces attribute is behind a feature flag?

Reply:
Usually these require a skipIfBeta call